setup.py

```diff
@@ -36,21 +37,27 @@ def read(*rnames):
           + '\n\n' +
           read('CHANGES.txt')
           ),
-      keywords = "zope3 skin zmi",
-      classifiers = [
+      keywords="zope3 skin zmi",
+      classifiers=[
           'Development Status :: 5 - Production/Stable',
           'Environment :: Web Environment',
           'Intended Audience :: Developers',
           'License :: OSI Approved :: Zope Public License',
           'Programming Language :: Python',
+          'Programming Language :: Python:: 2.7',
+          'Programming Language :: Python:: 3.5',
```

Comment:
I got to looking, and all of the packages I've opened Py 3 PRs for lately are [used by Plone](https://github.com/plone/Products.CMFPlone/issues/1351#issuecomment-178011652). Plone is not quite ready for Python 3 yet, but they will be, so there is good motivation to add 3.4 support (hopefully they're ready before 2019 😄 ). I've done so.